File: #5085/number15.php
<?php

$firstNumber = intval(trim(fgets(STDIN)));
$finalNumber = intval(trim(fgets(STDIN))) + 1;
$countEvenNumbers = 0;

while($firstNumber < $finalNumber) {
	if (!($firstNumber % 2)) {
		$countEvenNumbers++;
	}
	$firstNumber++;
}
echo $countEvenNumbers;
?>

File: #5085/number15.cpp
#include <iostream>

using namespace std;

int main() {
	int firstNumber, finalNumber, countEvenNumbers;

	cin >> firstNumber;
	cin >> finalNumber;

	finalNumber++;

	while (firstNumber < finalNumber) {
		if (!(firstNumber % 2)) {
			countEvenNumbers++;
		}
		firstNumber++;
	}
	cout << countEvenNumbers << endl;
	return 0;
}